examples/utils/testpage.js

```diff
@@ -6,8 +6,8 @@ export default function test(callback) {
         'body {font-family: monospace;}',
         'dt, dd {display: inline-block; margin: 0;}',
         'dt {min-width: 15em;}',
-      ].join('\n'),
-    ),
+      ].join('\n')
```

Comment:
Ooof, that is a real shame that we apparently need to change the prettier setting just for this one single trailing comma 😭